### ISSUE TITLE
Webpack build fails in 4.25.0

### ISSUE THREAD

**Issue by rdk8** [digitalfortress-tech/typeahead-standalone#77]
### What happened?

Webpack results in an error in 4.25.0:

Module not found: Error: Package path ./dist/basic.css is not exported from package /home/raja/dev/typeahead-standalone-debug/node_modules/typeahead-standalone (see exports field in /home/raja/dev/typeahead-standalone-debug/node_modules/typeahead-standalone/package.json)

### Affected version

v4.25.0, v4.25.1

### Reproduction URL

NA -- local build

### How to reproduce?

A minimal project to reproduce
1. npm init -y
2. npm install webpack webpack-cli typeahead-standalone  css-loader style-loader --save-dev
3. Add index.js and webpack.config.js as described below
4. npm run build

index.js:
```
import typeahead from 'typeahead-standalone';
import 'typeahead-standalone/dist/basic.css';

window.typeahead=typeahead;
```

webpack.config.js
```
const path = require('path');

module.exports = {
  entry: './src/index.js',
  output: {
    filename: 'bundle.js',
    path: path.resolve(__dirname, 'dist'),
  },
  module: {
    rules: [
      {
        test: /\.css$/,
        use: ['style-loader', 'css-loader'],
      },
    ],
  },
  mode: 'development',
};
```

package.json
```
{
  "name": "typeahead-standalone-debug",
  "version": "1.0.0",
  "main": "index.js",
  "scripts": {
    "build": "webpack"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "description": "",
  "devDependencies": {
    "css-loader": "^7.1.1",
    "style-loader": "^4.0.0",
    "typeahead-standalone": "4.25.0",
    "webpack": "^5.91.0",
    "webpack-cli": "^5.1.4"
  }
}
```

### Relevant log output

```Shell
> typeahead-standalone-debug@1.0.0 build
> webpack

asset bundle.js 19.3 KiB [emitted] (name: main)
runtime modules 670 bytes 3 modules
cacheable modules 14.1 KiB
  ./src/index.js 122 bytes [built] [code generated]
  ./node_modules/typeahead-standalone/dist/typeahead-standalone.es.js 14 KiB [built] [code generated]

ERROR in ./src/index.js 2:0-45
Module not found: Error: Package path ./dist/basic.css is not exported from package /typeahead-standalone-debug/node_modules/typeahead-standalone (see exports field in /typeahead-standalone-debug/node_modules/typeahead-standalone/package.json)

webpack 5.91.0 compiled with 1 error in 162 ms
```

### Additional context

The same process works if typeahead-standalone is fixed to 4.24.0

**Comment by niketpathak** [comment]
Thank you for reporting this bug. It has been fixed in **v4.25.5**

**Comment by rdk8** [comment]
Confirmed it works for me on 4.26.0.
